Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,92 @@ Meaning of columns:
+| `IfcCartesianTransformationOperator2DnonUniform` | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcCircle`                           | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCircularArcSegment2D`             | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcClothoid`                         | :x:                  | `CurveConverter`           | -                     | [#304](https://github.com/tumcms/Open-Infra-Platform/issues/304) |
+| `IfcCompositeCurve`                   | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCompositeCurveOnSurface`          | :x:                  | -                          | -                     | -     |
+| `IfcCompositeCurveSegment`            | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcCsgSolid`                         | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcCurveSegment`                     | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcDirection`                        | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcEllipse`                          | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcExtrudedAreaSolid`                | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcExtrudedAreaSolidTapered`         | :x:                  | -                          | -                     | -     |
+| `IfcFaceBasedSurfaceModel`            | :heavy_check_mark:   | `FaceConverter`            | -                     | -     |
+| `IfcFacetedBrep`                      | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcFacetedBrepWithVoids`             | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcFillAreaStyleHatching`            | :x:                  | -                          | -                     | -     |
+| `IfcFillAreaStyleTiles`               | :x:                  | -                          | -                     | -     |
+| `IfcFixedReferenceSweptAreaSolid`     | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcGeometricCurveSet`                | :x:                  | -                          | -                     | -     |
+| `IfcGeometricSet`                     | :heavy_check_mark:   | `RepresentationConverter`  | -                     | -     |
+| `IfcGradientCurve`                    | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcHalfSpaceSolid`                   | :clock9:             | `SolidModelConverter`      | -                     | -     |
```

Comment:
I believe this one's supported.

Reply:
The function with IfcHalfSpaceSolid was empty

Reply:
I see, the code is currently commented out. Then it's not supported.